src/toast/stack/_toast-stack.scss

```
@@ -0,0 +1,10 @@
+.toast-stack {
+  position: fixed;
+  bottom: 0;
+  left: calc(50vw - var(--toast-width) / 2);
+
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: center;
+  justify-content: center;
```

Let's just display them on the bottom right by default
```suggestion
  position: fixed;
  bottom: 0;
  right: 0;
  z-index: 2;

  max-height: 100vh;

  overflow: auto;

  padding: 24px;
```